docs/extensions.md

```diff
@@ -201,14 +202,31 @@ The client secret and tenant ID can be found in the "Overview" tab:
 4. Put in your Tenant ID in the authorization URL
 The URL is as such: `https://login.microsoftonline.com/TENANT_ID/oauth2/v2.0/authorize`. The Token URL is not strictly required for ID Token auth.
 ![image](https://user-images.githubusercontent.com/5719530/169712857-fe49855e-da3a-4dca-9fde-59bdc60eaa4d.png)
+
+5. In the "Token Configuration" tab on the left, click "Add Optional Claim." Select the token type as "ID" and choose "email" as the claim, then click "Add."
+![token_configuration](https://github.com/user-attachments/assets/2f3b62b9-23a6-4404-a6c6-2dfd796a0b2d)
 	
-5. Done! Click save and log out. Try your new login based on your Azure AD configuration. 
+7. Done! Click save and log out. Try your new login based on your Azure AD configuration. 
 	
-PS: When the user is signed in, they have the access rights of a "user" in the designated organization, and will have a username according to the ID decided in O365. This can be changed by admins.
+PS: When a user signs in, they are granted the access rights of a "user" within the designated organization. The username will be derived from the email address listed in the Azure users list. Therefore, when creating or adding a new user in Azure, ensure that the email field in their profile is populated. If this email field is empty, the user will not be able to log in to the organization.
 	
 ### Other Platforms
 As long as you can create an identity and acquire an Entrypoint (IdP) and X509, paste them into the Shuffle fields, and it should work with any SAML/SSO provider.
 
+### Testing SSO
```

Comment:
There is already a Testing SSO section. Please modify that one instead of adding another

Reply:
I removed previous "Testing SSO" section and keeping new testing sso section at the bottom of sso docs.